### DISCUSSION TITLE
how to i make explorerpatcher have a search bar with everything toolbar?

### DISCUSSION THREAD

**Discussion by Rezy774** [valinet/ExplorerPatcher#2428] [Q&A]
im on windows 11 build:  22621.2428.59.1 and explorerpatcher build: [22621.2428.59.1]

**Comment by pyrates999** [comment]
windows build 22621 is incomplete.  Please check it by running winver.

**Reply by Rezy774** [reply]
I’m on windows build 22621.2428.59.1

**Reply by Rezy774** [reply]
Now just tell me how to get the search box with everything toolbar
